api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/EmailMessagePayloadFactory.java

```diff
@@ -57,6 +58,18 @@ public EmailMessagePayload buildEmailMessagePayload(PaymentCycle paymentCycle, E
                 .build();
     }
 
+    public EmailMessagePayload buildEmailMessagePayloadWithFirstAndLastNameOnly(Claim claim, EmailType emailType) {
```

Comment:
This might be able to be refactored with the code in `DetermineEntitlementNotificationHandler`. Similarly for the unit test below with `DetermineEntitlementNotificationHandlerTest`.

Reply:
Moved the private method out of DetermineEntitlementNotificationHandler.